src/components/feed/BookmarksContainer.js

```diff
@@ -0,0 +1,23 @@
+import React, {Fragment} from 'react';
+import "./feed.css"
+import Alerts from '../Alerts';
+import BookmarkedPosts from '../posts/BookmarkedPosts';
+import { useDispatch } from 'react-redux/es/exports';
+import { useGetBookmarksQuery } from '../../services/extendedApi';
+import PostCard from '../posts/PostCard';
```

Comment:
you can use a combination of absolute path and barrels to avoid including `.` in your imports.

Reply:
Resolved. Thanks